quartz/components/Graph.tsx

```diff
@@ -53,10 +55,11 @@ const defaultOptions: GraphOptions = {
     showTags: true,
     removeTags: [],
     focusOnHover: true,
+    enableRadial: false,
```

Comment:
Should this be set to `true` to match the documentation? (radial on global graph, non-radial on local graph)